homeassistant/components/nmbs/__init__.py

```diff
@@ -1 +1,67 @@
-"""The nmbs component."""
+"""The NMBS component."""
+
+import logging
+
+from pyrail import iRail
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_STATION_FROM, CONF_STATION_LIVE, CONF_STATION_TO, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+PLATFORMS = [Platform.SENSOR]
+
+
+CONFIG_SCHEMA = cv.empty_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the NMBS component."""
+
+    api_client = iRail()
+
+    hass.data.setdefault(DOMAIN, {})
+    station_response = await hass.async_add_executor_job(api_client.get_stations)
+    if station_response == -1:
+        return False
+    hass.data[DOMAIN]["stations"] = station_response["station"]
```

Comment:
We're only storing the stations in here, we don't have to put the stations in a dict, we can just put it directly in hass.data

(Also, for a followup (or in this one if you feel like it), it would be nice if we could `HassKey` here to add typing to `hass.data`)